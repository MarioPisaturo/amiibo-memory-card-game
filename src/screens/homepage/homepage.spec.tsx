import * as React from 'react';
import { shallow } from 'enzyme';

import Homepage, { IHomepage } from './homepage';

describe('Homepage', () => {
  function makeProps(overrideProps: IHomepage = {}): IHomepage {
    return {
      ...overrideProps,
    };
  }

  it('should render a Homepage', () => {
    const props: IHomepage = makeProps();
    const wrapper = shallow(<Homepage {...props} />);
    expect(wrapper).toHaveLength(1);
  });

  it('Homepage should match snapshot', () => {
    const props: IHomepage = makeProps();
    expect(shallow(<Homepage {...props} />)).toMatchSnapshot();
  });
});

/** TODO: add other tests */
