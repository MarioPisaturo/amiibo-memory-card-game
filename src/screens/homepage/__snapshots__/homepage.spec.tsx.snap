// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage Homepage should match snapshot 1`] = `ShallowWrapper {}`;
